### ISSUE TITLE
Validation Occurs Even on Disabled Elements

### ISSUE THREAD

**Issue by homestar9** [guillaumepotier/Parsley.js#1343]
What kind of issue is this? (put 'x' between the square brackets)

 - [ ] Question. This issue tracker is not the place for questions. If you want to ask how to do
       something, or to understand why something isn't working the way you expect it to, use
       http://stackoverflow.com/questions/ask .
       Provide working code, starting from http://codepen.io/marcandre/pen/jqbzyN?editors=101.
       We monitor the tag `parsley.js`.

 - [x ] Bug report. If you’ve found a bug, you must provide a minimal example in a CodePen,
       starting from http://codepen.io/marcandre/pen/jqbzyN?editors=101 .

 - [ ] Feature Request. Make sure there's no good way to do what you want first;
       consider asking on http://stackoverflow.com/questions/ask first.

It looks like Parsley validates disabled form elements.  I suppose this issue may fall into the "feature" category if this behavior is intended, however, my gut tells me, that a disabled form field should not be validated.

Here's an example codepen:
https://codepen.io/homestar9/pen/NWrYYyZ

Here's an animation of the issue:
https://i.imgur.com/SUvfl99.gif

**Comment by marcandre** [comment]
You are right, Parsley should probably follow the standard here. I haven't changed this for compatibility, and I put in the [documentation about the 'excluded' option](http://parsleyjs.org/doc/index.html#usage-form) what should probably be the correct default.
